src/Analysim.Web/ClientApp/src/app/projects/project-overview/project-overview-view/project-content/modal-datasets/modal-datasets.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Notebook } from 'src/app/interfaces/notebook';
+import { Project } from 'src/app/interfaces/project';
+import { ProjectService } from 'src/app/services/project.service';
+
+@Component({
+  selector: 'app-modal-datasets',
+  templateUrl: './modal-datasets.component.html',
+  styleUrls: ['./modal-datasets.component.scss']
+})
+export class ModalDatasetsComponent implements OnInit {
+
+  isLoading: Boolean;
+  fileTree: any;
+
+  @Output() closeModal: EventEmitter<any> = new EventEmitter();
+
+  @Input() notebook: Notebook;
+  @Input() project: Project;
+
+  blobFiles = [
+    {
+      directory: "test/",
+      name: "test.txt",
+    },
+    {
+      directory: "test/hero/",
+      name: "hero.txt",
+    },
+    {
+      directory: "honda/",
+      name: "honda.txt",
+    }
+  ]
+
+  constructor(private projectService: ProjectService) { }
+
+  ngOnInit(): void {
+    console.log("received notebook in datasets : ", this.notebook);
```

Comment:
you can reduce the log after things are working. You can keep them commented out.

Reply:
done !!